cfn-resources/org-invitation/mongodb-atlas-orginvitation.json

```diff
@@ -120,18 +117,23 @@
       "description": "Email address of the MongoDB Cloud user invited to join the organization."
     }
   },
+  "required": [
+    "Profile"
+  ],
   "primaryIdentifier": [
-    "/properties/Id"
+    "/properties/Id",
+    "/properties/Profile"
   ],
   "readOnlyProperties": [
     "/properties/ExpiresAt",
-    "/properties/Id", 
+    "/properties/Id",
     "/properties/CreatedAt",
     "/properties/InviterUsername"
```

Comment:
have you tried to use GetAttr to retrieve these info in a stack? The example that you updated does not use `GetAttr`

Reply:
see this https://github.com/mongodb/mongodbatlas-cloudformation-resources/pull/340#discussion_r1114456753

Reply:
Fixed below